resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -418,6 +418,14 @@ package object config extends Logging {
       .stringConf
       .createOptional
 
+  private[spark] val INIT_CONTAINER_REMOTE_PYSPARK_FILES =
+    ConfigBuilder("spark.kubernetes.initcontainer.remotePyFiles")
```

Comment:
Good catch - I don't think this is necessary, this is an artifact of something I was trying before.